gitlabform/gitlabform/core.py

```diff
@@ -307,6 +308,28 @@ def process_branches(self, project_and_group, configuration):
                 if self.args.strict:
                     exit(3)
 
+    @if_in_config_and_not_skipped
+    def process_tags(self, project_and_group, configuration):
+        for tag in sorted(configuration['tags']):
+            try:
+                if configuration['tags'][tag]['protected']:
+                    create_access_level = configuration['tags'][tag]['create_access_level'] if \
+                        'create_access_level' in configuration['tags'][tag] else '30'
```

Comment:
I would not pass any value to GitLab's API if user won't set the `create_access_level` in gitlabform's config. Then GitLab's API default (`40`, according to https://docs.gitlab.com/ee/api/protected_tags.html#protect-repository-tags), will be used, which I think is what the user expects.

Reply:
the "exists"-check is done incorrectly:
- https://github.com/egnyte/gitlabform/pull/39#discussion_r566725404